README.md

```diff
@@ -66,10 +66,9 @@ and already the latest stable version:
 dart --version
 ```
 
-### Clone this repo and its submodules
+### Clone this repo
 
 > [!NOTE]
```

Comment:
I suggest we just git rid of this whole note; I don't think this README is the place to teach folks about git